.dialyzer-ignore.exs

```diff
@@ -0,0 +1,3 @@
+[
+  {":0:unknown_type Unknown type: Mongo.WriteError.t/0." }
+] 
```

Comment:
New lines at the end of the file are nice to have (and required by POSIX)

Reply:
Made the change that is required.